recipes/piexif/meta.yaml

```
@@ -0,0 +1,63 @@
+{% set name = "Piexif" %}
+{% set version = "1.1.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/hMatoba/Piexif/archive/1.1.2.tar.gz
+  sha256: 1fd0e5d21e9a69be99205979b927b1803c2daae06a4e54a802b0d0d520932722
+
+build:
+
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+  host:
+    - python
+    - pip
+    - Pillow==5.4.1
+    - Sphinx
+    - sphinxcontrib-websupport
+  run:
+    - python
+    - Pillow==5.4.1
```

You may want this pin to be less strict
```suggestion
    - pillow =5.4.1
```